.czrc

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@endemolshinegroup/cz-jira-smart-commit",
+  "@endemolshinegroup/cz-jira-smart-commit": {
+    "pullRequestsOnly": true,
+    "scopes": {
+      "scss": "For changes in scss files"
```

Comment:
Nice! You could take a look at the other repositories and add default scopes from there too, like "cz" or "dependencies"

Reply:
ok